plugin/storage/grpc/config/config.go

```diff
@@ -122,6 +122,9 @@ func (c *Configuration) buildPlugin(logger *zap.Logger) (*ClientPluginServices,
 		HandshakeConfig: shared.Handshake,
 		VersionedPlugins: map[int]plugin.PluginSet{
 			1: shared.PluginMap,
+			2: map[string]plugin.Plugin{
```

Comment:
It's been a while since I looked into this, could you elaborate what `VersionedPlugins` does?

Reply:
VersionedPlugins is used in go-plugins to negotiate plugin protoversion between servers and clients. Go-plugin will find the latest version supported by both the server and the client, and start the plugin server. 

In this scenario, it is more reasonable to negotiate with Capabilities, I'll refactor the code.